src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -411,20 +456,16 @@ Namespace Microsoft.CodeAnalysis.Operations
         End Function
 
         Private Function GetVariableDeclarator(boundLocalDeclaration As BoundLocalDeclaration) As IVariableDeclaratorOperation
-            Dim initializer As Lazy(Of IVariableInitializerOperation) = New Lazy(Of IVariableInitializerOperation)(
-                Function()
-                    If boundLocalDeclaration.IdentifierInitializerOpt IsNot Nothing Then
-                        Dim syntax = boundLocalDeclaration.Syntax
-                        Dim initializerValue As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundLocalDeclaration.IdentifierInitializerOpt))
-                        Return New LazyVariableInitializer(initializerValue, _semanticModel, syntax, type:=Nothing, constantValue:=Nothing, isImplicit:=True)
-                    Else
-                        Return Nothing
-                    End If
-                End Function)
-            Dim ignoredArguments As Lazy(Of ImmutableArray(Of IOperation)) = New Lazy(Of ImmutableArray(Of IOperation))(
-                Function() ImmutableArray(Of IOperation).Empty)
+            Dim initializer As IVariableInitializerOperation = Nothing
+            If boundLocalDeclaration.IdentifierInitializerOpt IsNot Nothing Then
+                Dim syntax = boundLocalDeclaration.Syntax
+                Dim initializerValue As BoundNode = boundLocalDeclaration.IdentifierInitializerOpt
+                initializer = New VisualBasicLazyVariableInitializerOperation(Me, initializerValue, _semanticModel, syntax, type:=Nothing, constantValue:=Nothing, isImplicit:=True)
+            End If
+
+            Dim ignoredArguments = ImmutableArray(Of IOperation).Empty
 
-            Return New LazyVariableDeclarator(boundLocalDeclaration.LocalSymbol, initializer, ignoredArguments, _semanticModel, boundLocalDeclaration.Syntax, type:=Nothing, constantValue:=Nothing, isImplicit:=boundLocalDeclaration.WasCompilerGenerated)
+            Return New VariableDeclaratorOperation(boundLocalDeclaration.LocalSymbol, initializer, ignoredArguments, _semanticModel, boundLocalDeclaration.Syntax, type:=Nothing, constantValue:=Nothing, isImplicit:=boundLocalDeclaration.WasCompilerGenerated)
```

Comment:
>VariableDeclaratorOperation [](start = 23, length = 27)

Creating not lazy node?

Reply:
There is no initializer for this node, so there are no children that could be created.

---
In reply to: [232415458](https://github.com/dotnet/roslyn/pull/31010#discussion_r232415458) [](ancestors = 232415458)